app/src/main/java/com/lagradost/cloudstream3/utils/DataStoreHelper.kt

```diff
@@ -131,7 +131,6 @@ object DataStoreHelper {
 
                             // update UI
                             setAccount(getDefaultAccount(context), true)
-                            MainActivity.bookmarksUpdatedEvent(true)
```

Comment:
NOTE: This is removed because it is also done in setAccount so it was doing it twice. If that was incorrect, I will revert this part.